UnzipFile - error (#23745)
demisto/content: Packs/CommonScripts/ReleaseNotes/1_10_47.md

#### Scripts
##### UnzipFile

> - Fixes an issue where warning were being returned as error.e *FiltersAndTransformers* pack.

Reviewer: ```suggestion
Fixed an issue where warnings were returned as errors when running this script on certain files.
```
